packages/reusable-blocks/src/components/reusable-blocks-menu-items/reusable-block-convert-button.js

```diff
@@ -191,8 +191,7 @@ export default function ReusableBlockConvertButton( {
 							/>
 							<HStack justify="right">
 								<Button
-									// TODO: Switch to `true` (40px size) if possible
-									__next40pxDefaultSize={ false }
+									__next40pxDefaultSize
```

Comment:
This isn't used in the app anymore, but we can infer that it's safe from the similar UI patterns like this:

![Add new pattern popover](https://github.com/user-attachments/assets/7828d832-64b9-46db-9ae4-b40ca5d39525)